libraries/helpers.rb

```diff
@@ -160,6 +160,39 @@ def ami_bootstrapped?
   version != '' && ( node['cfncluster']['skip_install_recipes'] == 'yes' || node['cfncluster']['skip_install_recipes'] == true )
 end
 
+#
+# Check if OS type specified by the user is the same as the OS identified by Ohai
+#
+def validate_OS_type
```

Comment:
Minor: Let's try to name things in `snake_case`(lowercase) according to [this guide](https://github.com/rubocop-hq/ruby-style-guide#snake-case-symbols-methods-vars)